src/test/resources/xtdb/sql/logic_test/direct-sql/period_predicates.test

```diff
@@ -36,13 +36,6 @@ SELECT t1.xt$id, t1.start, t1.end FROM t1 WHERE PERIOD (t1.start, t1.end) PRECED
 2000-01-01
 2020-01-01
 
-query ITT nosort

```

Comment:
at a glance this one still seems relevant?

Reply:
This has to do with the `nil` system-to issue. The predicate doesn't work on that.